frontend/public/components/storage/create-pvc.tsx

```diff
@@ -13,19 +13,24 @@ import { Checkbox } from '../checkbox';
 import { PersistentVolumeClaimModel } from '../../models';
 import { StorageClass } from '../storage-class-form';
 import {
-  cephRBDProvisionerSuffix,
   provisionerAccessModeMapping,
   initialAccessModes,
   accessModeRadios,
   volumeModeRadios,
   dropdownUnits,
   getAccessModeForProvisioner,
+  getVolumeModeForProvisioner,
 } from './shared';
 
-const InfoToolTip = () => (
+const InfoToolTip = (props) => (
```

Comment:
InfoToolTip is our own function, and it uses <Tooltip> which already comes from the library only, It is not necessary to call this function, we can directly use <Tooltip>

Reply:
Yes but still it would best to make it accessible from a common point. We are using it in bucket class page, pvc page and will most likely be using it in restore pvc modal also. It would avoid the trouble of creating it in every other page.

Reply:
No, it does not make any sense, the way we are using it here wrong. we don't have to add one more layer on top of  Tooltop, no need to call this InfoToolTip

Reply:
if we are combining more than one element and come up with the single jsx element then it makes sense, for example checkbox (label + input)

Reply:
Yes exactly ! We are using `<InfoToolTip> = <Tooltip /> + <OutlinedQuestionCircleIcon />` so that when someone hovers over the `?` icon, i can view the information.  We need to try to make this information customizable in other areas. 

If we don't make it generic, We will have to add the icon and tooltip specifically wherever we want to use it . I hope it makes sense

Reply:
Ah! ok i never saw OutlinedQuestionCircleIcon